package-lock.json

```diff
@@ -0,0 +1,10806 @@
+{
```

Comment:
why do you need this file? afaik `yarn` has been used on this project

Reply:
so if you added or removed some packages i assume that it should be reflected in yarn.lock not in package-lock.json